### ISSUE TITLE
does not work with project scoped errors

### ISSUE THREAD

**Issue by grosser** [stve/airbrake-api#37]
```
AirbrakeAPI.configure(xxx)
AirbrakeAPI.errors # works
AirbrakeAPI.errors(project_id: foo)
Kaboom ... returns weird html instead of xml :/
```

@stve time to switch to api v3 ?

**Comment by stve** [comment]
@grosser wow, they've updated nearly the entire API again. JSON only and reasonable docs now as well. Doesn't look like this will take me too long to update. I should be able to knock out an updated version this week/weekend.

**Comment by grosser** [comment]
yeah currently doing raw calls ... looks like pagination does not work :D

**Comment by grosser** [comment]
FYI most of the commands / endpoints you will need: https://github.com/jcheatham/airbrake_tools/pull/31

**Comment by stve** [comment]
thanks @grosser i'll keep you posted
